tests/network/vmi_networking.go

```diff
@@ -766,20 +781,32 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 				tcpPort := 8080
 				tests.StartPythonHttpServer(serverVMI, tcpPort)
 
-				assert.XFail("https://github.com/kubevirt/kubevirt/issues/5113", func() {
-					By("Checking ping (IPv6) from vmi to cluster nodes gateway")
-					// Cluster nodes subnet (docker network gateway)
-					// Docker network subnet cidr definition:
-					// https://github.com/kubevirt/project-infra/blob/master/github/ci/shared-deployments/files/docker-daemon-mirror.conf#L5
-					Expect(libnet.PingFromVMConsole(serverVMI, "2001:db8:1::1")).To(Succeed())
-				})
-
 				Expect(verifyClientServerConnectivity(clientVMI, serverVMI, tcpPort, k8sv1.IPv6Protocol)).To(Succeed())
 			},
 				table.Entry("with a specific port number [IPv6]", []v1.Port{{Name: "http", Port: 8080}}, ""),
 				table.Entry("without a specific port number [IPv6]", []v1.Port{}, ""),
 				table.Entry("with custom CIDR [IPv6]", []v1.Port{}, "fd10:10:10::/120"),
 			)
+
+			It("[outside_connectivity]should be able to reach the outside world [IPv6]", func() {
+				ipv6Address := "2001:db8:1::1"
```

Comment:
maybe name the var with some value acceding the comment on line 807?
the comment should also be near the var i think

Reply:
I think that in this context It will be just more confusing, since the user now can also change this address to some arbitrary IPV6 address she/he chooses( and not necessarily the docker network gateway) 

I agree that moving the comment closer to the default value will make it clear why this address was chosen as the default. I move it above the line where the default it set.